chat/chat-api/api/src/java/org/sakaiproject/chat2/model/TransferableChatMessage.java

```diff
@@ -0,0 +1,95 @@
+package org.sakaiproject.chat2.model;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Date;
+
+import lombok.Data;
+
+/**
+ * 
+ * Class used to transfer data through jGroups
+ *
+ */
+@Data
+public class TransferableChatMessage implements Serializable {
+
+	public enum MessageType {
+	    CHAT,
+	    HEARTBEAT,
+	    CLEAR,
+	    REMOVE;
+	}
+	
+	public MessageType type;
+	public String id;
+	public String owner;
+	public String siteId;
+	public String channelId;
+	public String content;
+	public long timestamp;
```

Comment:
Maybe it's a good idea change the ChatMessage.messageDate property from java.util.Date to java.time.Instant (or java.time.ZonedDateTime) to avoid problems saving in DB when cluster nodes are in different Timezones. But seems that, hibernate 4 does not support well these types.

We should move this to a different PR.